src/Symfony/Component/DependencyInjection/Container.php

```diff
@@ -282,7 +282,7 @@ public function get($id, $invalidBehavior = self::EXCEPTION_ON_INVALID_REFERENCE
         // calling $this->normalizeId($id) unless necessary.
         for ($i = 2;;) {
             if (isset($this->privates[$id])) {
-                @trigger_error(sprintf('Requesting the "%s" private service is deprecated since Symfony 3.2 and won\'t be supported anymore in Symfony 4.0.', $id), E_USER_DEPRECATED);
+                @trigger_error(sprintf('The "%s" service is private, getting it from the container is deprecated since Symfony 3.2 and will fail in 4.0. You should either make the service public or stop getting services directly from the container and use any dependency injection methods instead.', $id), E_USER_DEPRECATED);
```

Comment:
It's "any method", not "any methods" because only one of the methods should be used, not several at once.

Reply:
fixed